code/game/objects/items/storage/firstaid.dm

```diff
@@ -211,7 +211,7 @@
 	use_to_pickup = 1
 	storage_slots = null
 	use_sound = null
-	max_storage_space = 16
+	max_storage_space = 12
```

Comment:
change this back, maths is better for pill batching with 16

Reply:
8 or 16, agreed.